README.md

```diff
@@ -1,12 +1,16 @@
 # pfpk
 
-pic for private key. A [Cloudflare
+pic for public key. A [Cloudflare
 Worker](https://developers.cloudflare.com/workers) that allows associating a
-name and [Stargaze](https://stargaze.zone) or [Juno](https://junonetwork.io) NFT
-with a given [Cosmos](https://cosmos.network) wallet / keypair.
+name and NFT (image) with a given set of [Cosmos](https://cosmos.network)
+wallets / public keys.
 
 Currently deployed at https://pfpk.daodao.zone
 
+A profile contains a name and NFT image and is associated with one or more
+public keys. A chain preference must be set for each public key in order for it
```

Comment:
What is the purpose of the chain preference? This seems like a reasonable place to explain the need for a chain preference.